src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidatorTest.java

```diff
@@ -290,4 +290,10 @@ public void shouldNotFailIfMaxLatencyTier2andMaxLatencyAreNull() throws InvalidF
         RequestedFlow flow = getTestRequestWithMaxLatencyAndMaxLatencyTier2(null, null);
         flowValidator.checkMaxLatency(flow);
     }
+
+    @Test
+    public void shouldNotFailIfMaxLatencyTier2andMaxLatencyAreEqual() throws InvalidFlowException {
```

Comment:
I would add a similar test to YFlowValidatorTest to be consistent. Or test it from ValidatorUtils directly.

Reply:
sure